file: Send correct information to TM during prepare

KristianIvarsson opened the pull request:
Without this fix casual-file-manager cannot be used with other RM's

lazan commented on middleware/file/source/manager/resource.cpp:
```diff
@@ -211,6 +211,20 @@ namespace casual::file::resource
+                  return reply;
+               }
+            } // reverse
+         } // xa
```
nitpick: `namespace xa::reverse` .
Otherwise, nice to have an abstraction for the 3 xa messages.

lazan reviewed (changes requested):
Nice! But we need a commit message that describe what the problem was, and how it was fixed. 
For me I find it easiest to write with the "convention":  
Before: bla bla
bla bla
Now: bla bla
bla bla

KristianIvarsson committed:
file: Send correct information to TM during prepare

Before this fix it wasn't possible to use file RM in conjunction with
other RM's

After this fix it is possible to use casual-file with other RM's

lazan reviewed (approved):
I still think the commit message doesn't tell me enough about what has been fixed... But, it could be worse...
